src/components/FlowChart/CommandBar.tsx

```diff
@@ -1,13 +1,36 @@
 import React from "react";
-import { ButtonGroup, Button } from "react-bootstrap";
+import styled from "@emotion/styled";
+import Play from "../../assets/icons/Play.svg";
+import Save from "../../assets/icons/Save.svg";
+import Stop from "../../assets/icons/Stop.svg";
+import Upload from "../../assets/icons/Upload.svg";
+import Yaml from "../../assets/icons/Yaml.svg";
 
 type Props = {
   importChart: () => void;
   copyChart: () => void;
   exportImage: (format: string) => void;
 };
 
-export default function ChartNode({
+const ButtonGroup = styled.div`
+  height: 3.125rem;
+  background: #009999;
+  border-radius: 8px;
+  display: flex;
+`;
+
+const Button = styled.button`
+  background: #009999;
+  border: 0;
+  margin-right: 1rem;
+`;
+
+const PlayButton = styled.button`
+  background: #009999;
+  border: 0;
+  margin: 0 2rem;
```

Comment:
All the buttons should have `cursor: pointer;` style.`Play` and `Stop` do not for some reason. Also `Play` should probably have same width as the other buttons.

Reply:
Also do the buttons look a bit big compared to other buttons in the app or is it just me?

Reply:
Play is wider in the Figma.

The Commandbar has the same height as the Pod Buttons. 
We can deploy it and then ask Lucia and Agnes about how they like it and then make changes